plugins/telescope/extensions/media-files.nix

```diff
@@ -0,0 +1,120 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+  (
+    import ./_helpers.nix {inherit lib helpers config pkgs;}
+  )
+  .mkExtension {
+    name = "media-files";
```

Comment:
I get `error: The option `plugins.telescope.extensions.media_files' does not exist.`

I think this should've had a rename deprecation added.

Reply:
Was it necessary to rename it to `media-files`... Not sure now.